src/interfaces/account.ts

```diff
@@ -2,6 +2,11 @@ import { Network } from './network'
 
 export type AccountId = string
 
+export type AccountPreferences = {
+  label: string
+  pfp: string // URL (https, ipfs or nft721://contractAddr/tokenId)
```

Comment:
I'm pretty sure that the comment isn't entirely correct as `pfp` currently stores the account address